x/service/module/tx_add_service_test.go

```diff
@@ -81,6 +84,15 @@ func TestCLI_AddService(t *testing.T) {
 			supplierAddress: account.Address.String(),
 			service:         svc1,
 		},
+		{
+			desc:            "valid - add new service without compute units per relay",
```

Comment:
Shouldn't we make this invalid?

Reply:
Looked at the coce. 

Please update test description to explain that it's valid because it defaults to the global value.